local_planner/src/nodes/planner_functions.cpp

```diff
@@ -105,13 +90,9 @@ void filterPointCloud(
   }
 
   if (counter_sphere > 0) {
-    temp_sphere_center.x = temp_sphere_center.x / counter_sphere;
-    temp_sphere_center.y = temp_sphere_center.y / counter_sphere;
-    temp_sphere_center.z = temp_sphere_center.z / counter_sphere;
+    temp_sphere_center /= counter_sphere;
```

Comment:
I'm not familiar enough with the Eigen library, but this looks suspicious. Do you indeed have to call the `array()` method for the += operator, but not for the /= ? Why?